packages/app-form-builder/src/page-builder/admin/plugins/components/PeFormElement.tsx

```diff
@@ -34,7 +34,8 @@ const PeForm: FormRenderer = props => {
                     } catch {
                         return apolloClient
                             .query({ query: gql(GET_PUBLISHED_FORM), variables })
-                            .then(({ data }) => data.formBuilder.getPublishedForm.data);
+                            .then(({ data }) => data.formBuilder.getPublishedForm.data)
+                            .catch(() => null);
```

Comment:
If add a form on the page and then we delete that form we get error `Form "64ee005792862d000941dc7a#0003" was not found!`. on the `Page Builder Editor` page. This is because the query in `catch` block is trying to query a FORM that does not exist. And the `catch` helps us to avoid that error.

Reply:
@adrians5j also It was a quick fix, I will imrove it